lib/src/tooltip_widget.dart

```diff
@@ -259,57 +274,106 @@ class _ToolTipWidgetState extends State<ToolTipWidget>
+                                                          .textTheme
+                                                          .subtitle2!
+                                                          .merge(
+                                                            TextStyle(
+                                                              color: widget
+                                                                  .textColor,
+                                                            ),
+                                                          ),
+                                                )
+                                              : SizedBox.shrink(),
+                                        ],
+                                      ),
+                                    ),
+                                    if (widget.actions != null)
+                                      SizedBox(
+                                        height: widget
+                                                .spaceBetweenTitleAndButtons ??
+                                            5,
                                       ),
-                                    ],
-                                  ),
+                                    if (widget.actions != null)
+                                      LayoutBuilder(
+                                          builder: (context, constraints) {
+                                        return Container(
+                                          color: widget.buttonsContainerColor ??
+                                              Colors.white,
+                                          height:
+                                              widget.buttonsContainerHeight ??
+                                                  40,
+                                          width: GetWidth(
+                                            key1: _key1,
+                                            key2: _key2,
+                                            buttonWidth:
+                                                widget.buttonsContainerWidth,
+                                            contentPadding:
+                                                widget.contentPadding,
+                                          ).getButtonsContainerWidth(),
+                                          child: widget.actions!,
+                                        );
+                                      }),
+                                  ],
                                 ),
                               ),
                             ),
                           )
-                        : widget.container!,
-                    if (widget.actions != null) widget.actions!
+                        : Column(
+                            crossAxisAlignment: CrossAxisAlignment.start,
+                            children: [
+                              widget.container!,
```

Comment:
widget.container should not be forced unwrap

Reply:
in this case it will not null as we already added condition